templates/virtualization-controller/_helpers.tpl

```diff
@@ -35,6 +35,24 @@
 {{- if (hasKey .Values.virtualization "virtualImages") }}
 - name: VIRTUAL_IMAGE_STORAGE_CLASS
   value: {{ .Values.virtualization.virtualImages.storageClassName }}
+- name: VIRTUAL_IMAGE_DEFAULT_STORAGE_CLASS
+  value: {{ .Values.virtualization.virtualImages.defaultStorageClassName }}
+- name: VIRTUAL_IMAGE_ALLOWED_STORAGE_CLASSES
+  value:
+    matchNames:
+{{- range .Values.virtualization.virtualImages.allowedStorageClassSelector.matchNames }}
+    - {{ . | quote }}
+{{- end }}
+{{- end }}
```

Comment:
This will not work for envs, value is always a string. The template should be as before: `join ","`. Same is for `VIRTUAL_DISK_ALLOWED_STORAGE_CLASSES` env.

Reply:
Fixed.